nifi-nar-bundles/nifi-atlas-bundle/nifi-atlas-reporting-task/src/test/resources/server.properties

```diff
@@ -21,25 +22,27 @@ broker.id=0
 
 ############################# Socket Server Settings #############################
 
-# The port the socket server listens on
+# The address the socket server listens on. It will get the value returned from 
+# java.net.InetAddress.getCanonicalHostName() if not configured.
+#   FORMAT:
+#     listeners = listener_name://host_name:port
+#   EXAMPLE:
+#     listeners = PLAINTEXT://your.host.name:9092
+#listeners=PLAINTEXT://:9092
 port=9092
 
-# Hostname the broker will bind to. If not set, the server will bind to all interfaces
-#host.name=localhost
-
-# Hostname the broker will advertise to producers and consumers. If not set, it uses the
-# value for "host.name" if configured.  Otherwise, it will use the value returned from
-# java.net.InetAddress.getCanonicalHostName().
-#advertised.host.name=<hostname routable by clients>
+# Hostname and port the broker will advertise to producers and consumers. If not set, 
```

Comment:
Minor: In the previous section `address` was used, here `Hostname and port`. Bit confusing (are these different concepts?)
Could use the same (either of the two) at both places.

Reply:
See my comment about Kafka `server.properties` above.